src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -29,31 +29,40 @@ export const CentralImportDuplicateSearch = ({ duplicateSearchFilters }: Central
   duplicateSearchFilters.yearPublished.length > 0 &&
     queryArray.push(`${DescriptionFieldNames.PublicationYear}:"${duplicateSearchFilters.yearPublished}"`);
 
-  const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')})` : '';
+  const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')}&results=${maxHits})` : '';
 
-  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
-    url: `${SearchApiPath.Registrations}?${searchQuery}&results=${maxHits}`,
-    errorMessage: t('feedback.error.search'),
+  const importCandidateQuery = useQuery({
+    queryKey: ['importCandidates', searchQuery],
+    queryFn: () => fetchImportCandidates(),
```

Comment:
liten forenkling, og unngår å instansiere en ekstra funksjon per render
```suggestion
    queryFn: fetchImportCandidates,
```